ideas/310-core-team-proposal.md

```diff
@@ -1,12 +1,21 @@
 ---
-title: Status Core Team
-
+id: #310-core-team-proposal
```

Comment:
Can we remove the proposal from id? #310-core-team or even just #310-core

Reply:
done